sw/airborne/modules/datalink/extra_pprz_dl.c

```diff
@@ -47,13 +47,25 @@
 #include "subsystems/datalink/telemetry.h"
 
 
+
+bool extra_dl_msg_available;
+uint8_t extra_dl_buffer[MSG_SIZE]  __attribute__((aligned));
```

Comment:
This way we have one buffer for *regular* datalink messages, and one for the *extra* datalink messages. If we don't use the extra buffer, the two streams would have to share the same buffer. Does it make sense?